<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>login</class>
    <widget class="QWidget" name="login">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>UBUVoiceAssistant</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QComboBox" name="dropLang">
                <property name="geometry">
                    <rect>
                        <x>560</x>
                        <y>20</y>
                        <width>221</width>
                        <height>25</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLabel" name="img">
                <property name="geometry">
                    <rect>
                        <x>110</x>
                        <y>80</y>
                        <width>533</width>
                        <height>357</height>
                    </rect>
                </property>
                <property name="text">
                    <string />
                </property>
                <property name="pixmap">
                    <pixmap>../../imgs/UBUVoiceAssistant_logo.png</pixmap>
                </property>
            </widget>
            <widget class="QLabel" name="lblUser">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>450</y>
                        <width>91</width>
                        <height>17</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#UserLabel</string>
                </property>
            </widget>
            <widget class="QLabel" name="lblPassword">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>480</y>
                        <width>121</width>
                        <height>17</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#PasswordLabel</string>
                </property>
            </widget>
            <widget class="QLabel" name="lblHost">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>510</y>
                        <width>101</width>
                        <height>17</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#HostLabel</string>
                </property>
            </widget>
            <widget class="QLineEdit" name="tbxUser">
                <property name="geometry">
                    <rect>
                        <x>280</x>
                        <y>450</y>
                        <width>151</width>
                        <height>25</height>
                    </rect>
                </property>
            </widget>
            <widget class="QLineEdit" name="tbxPassword">
                <property name="geometry">
                    <rect>
                        <x>280</x>
                        <y>480</y>
                        <width>151</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="echoMode">
                    <enum>QLineEdit::Password</enum>
                </property>
            </widget>
            <widget class="QLineEdit" name="tbxHost">
                <property name="geometry">
                    <rect>
                        <x>280</x>
                        <y>510</y>
                        <width>151</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="placeholderText">
                    <string>https://ubuvirtual.ubu.es</string>
                </property>
            </widget>
            <widget class="QCheckBox" name="chkUser">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>450</y>
                        <width>191</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#chkRememberUser</string>
                </property>
            </widget>
            <widget class="QCheckBox" name="chkHost">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>510</y>
                        <width>181</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#chkRememberHost</string>
                </property>
            </widget>
            <widget class="QPushButton" name="btnLogin">
                <property name="geometry">
                    <rect>
                        <x>370</x>
                        <y>550</y>
                        <width>83</width>
                        <height>25</height>
                    </rect>
                </property>
                <property name="text">
                    <string>#btnLogin</string>
                </property>
            </widget>
        </widget>
        </widget>
    <resources />
    <connections />
</ui>
